app/helpers/lightweight_activity_helper.rb

```diff
@@ -58,4 +58,13 @@ def complete_badge_for(activity)
       return ribbon(t("COMPLETED"),"my-ribbon")
     end
   end
+
+  def activity_player_path(activity)
+    activity_api_path = "#{api_v1_activity_url(@activity.id)}.json"
+    return  "#{ENV['ACTIVITY_PLAYER_URL']}/?activity=#{CGI.escape(activity_api_path)}"
```

Comment:
By using `_path` it won't be a full URL. So then this won't work well when we are testing on authoring.staging.concord.org or lara-master.concordqa.org and production since those will be different domains.

Reply:
It's grabbing `api_v1_activity_url`. It's the variable that is (maybe poorly-) named `_path`.

Reply:
I want ahead and renamed the variable (and helper methods) just to make that totally clear.

Reply:
Sorry, I didn't see that before. Thanks for updating it.